### ISSUE TITLE
setupLegacyFocus or setupLegacyFocusClasser?

### ISSUE THREAD

**Issue by chucklorenz** [cgkineo/adapt-cover-menu#5]
adapt-cover.js (master line 67; v2 line 81) reference setupLegacyFocus but I cannot locate that function. I did, however, locate setupLegacyFocusClasser in accessibility.js. Is this a typo?

**Comment by chris-steele** [comment]
@chucklorenz like issue #4, this is orphaned and can (and will) be removed.
